lib3/yaml/constructor.py

```diff
@@ -5,7 +5,8 @@
 from .error import *
 from .nodes import *
 
-import collections, datetime, base64, binascii, re, sys, types
+import datetime, base64, binascii, re, sys, types
+from collections import abc
```

Comment:
👍Bye-bye Python2 👍

Reply:
Please explain the abc change

Reply:
It's been cherry-picked from another PR. Basically, over time part of entities migrated from `collections` to `collections.abc` preserving backward compatibility but then the old location has been deprecated and it just crashes the program (Python 3.7+) instead of the deprecation warning (Python 3.6-).

Change in this file only affects Python 3, not Python 2 (which only has the old location).

Reply:
https://docs.python.org/3/library/collections.abc.html?highlight=abc#module-collections.abc

Reply:
Just to be sure here... abc doesn't add an external module dependency to pyyaml, right?

Reply:
Nope :) https://docs.python.org/3/library/collections.abc.html

Reply:
I believe the correct 2&3 solution is:
```py
try:
    from collections.abc import Hashable
except ImportError:
    from collections import Hashable
```

Reply:
See my PR for details and an explanation: https://github.com/yaml/pyyaml/pull/181

Since this is in `lib3/` which is used with Python 3 only, I don't see why there should be Python 2 compatibility code here.

Reply:
@dimaqq I was about to comment the same as @The-Compiler just did :)

Reply:
So sorry for the false alarm!